package.json

```diff
@@ -1,8 +1,10 @@
 {
   "name": "stellar-account-viewer-v2",
-  "prettier": "@stellar/prettier-config",
   "version": "1.3.0",
   "license": "Apache-2.0",
+  "engines": {
+    "node": ">=18.0.0"
+  },
   "lint-staged": {
     "src/**/*.ts?(x)": [
       "eslint --fix --max-warnings 0"
```

Comment:
do the `scripts` commands need to be updated to use `pnpm`?

Reply:
:100: Yes, we'll need to update all instances of `yarn` (scripts, readme, etc.)